### PR TITLE
Add handling of low-res screens in some components

### PR DESCRIPTION
**Pull request by jansule**
### Description:

Added media queries to a few components to make them appear smaller on low resolution screens. This  includes `ToggleButton`, `SimpleButton`, `Titlebar` and `Toolbar`. Also added a class to the modal that opens when adding a label in `DigitizeButton`.

### CODE REVIEW

File: src/Button/SimpleButton/SimpleButton.less [hunk 1]
```diff
@@ -19,4 +19,36 @@ button.react-geo-simplebutton {
...
+
+      span {
+        font-size: 2.5em;
+        line-height: 22px;
+      }
+    }
+  }
+}
+
+button.react-geo-togglebutton {
```
Comment (dnlkoch) [inline src/Button/SimpleButton/SimpleButton.less]: The same definition is also available in the `ToggleButton.less`. Is this intended?

File: src/Button/SimpleButton/SimpleButton.less [hunk 2]
```diff
@@ -19,4 +19,36 @@ button.react-geo-simplebutton {
     background-color: @component-background;
     border-color: darken(@component-background, 10);
   }
+
+  @media screen and (max-width: 1280px) {
```
Comment (dnlkoch) [inline src/Button/SimpleButton/SimpleButton.less]: The threshold value of `1280px` could possibly be moved to a less variable (see [here](https://github.com/terrestris/react-geo/blob/master/src/style/variables.less)).

File: src/Button/SimpleButton/SimpleButton.less (hunk 1 above)
Comment (jansule) [reply src/Button/SimpleButton/SimpleButton.less]: No. Thanks for the hint. Copy&Paste error. Will be resolved with next commit.

**Comment by marcjansen** [comment]
Can you provide some screenshots before and after? On those screens? Thx

**Comment by marcjansen** [comment]
Also, does this somehow clash with the widths of some popular smartphones? I.e. is mobile display equally good before and after?

**Comment by jansule** [comment]
@ahennr and I took a look at the quality on smartphone screens. They are affected by this change, but still working fine. Buttons are still large enough.

**Comment by jansule** [comment]
**Before:**
image

**After:**
image

**Comment by weskamm** [comment]
Usually we want bigger buttons on mobiles, as they are harder to use when too small.
Not sure if i am mixing things up?

**Comment by jansule** [comment]
Talked to @weskamm and @ahennr and we came to the conclusion that it does not make sense to make these changes directly in react-geo but rather within the respective projects. Also as @weskamm pointed out, reducing the button sizes is counterproductive for mobile development.

Closing PR.

**Pull request closed without merging**